Util/src/main/java/io/deephaven/util/ExecutionContext.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.util;
```

Comment:
Maybe this should be somewhere higher up? Maybe introduce a new module `engine-context` lower than `engine-updategraph`, in a package called `io.deephaven.engine.context`? Maybe move compiler tools up to that package, too?